### ISSUE TITLE
Support for boxplots

### ISSUE THREAD

**Issue by r2o2** [Brunel-Visualization/Brunel#165]
It would be useful to have support for boxplots in Brunel. It would probably be easiest to use as a new element, so you could easily create clustered boxplots:

`boxplot x(jobrole, gender) y(salary) color(gender)`

**Comment by cjiang-ibm** [comment]
YES, WML Visualization team needs this too!

**Comment by lkitt** [comment]
We would like a boxplot that shows the entire range from min to max, if that is an option.

**Comment by danrope** [comment]
Do you mean a range chart?

`data('sample:US States.csv') bar x(Region) yrange(Summer) range(Summer) `

**Comment by lkitt** [comment]
I mean we want the first option on the following page, rather than the second option.

http://www.physics.csbsju.edu/stats/box2.html

**Comment by danrope** [comment]
Try:

`data('sample:US States.csv') bar x(Region) yrange(Summer) range(Summer) style("width:1px") +   bar x(Region) yrange(Summer) iqr(Summer) +  point x(Region) y(Summer) median(Summer) style('symbol:rect;height:2')`

Styles can be improved here.  Can also probably add the hashes at the min/max if needed.

**Comment by lkitt** [comment]
Yes that's what I was looking for, thanks Dan.  If we can get clustering working then we will have what I want.

**Comment by GraemeHutcheon** [comment]
I would also like this - pretty standard chart for the team we present data to - Dan your example is close but really need to also show the outliers.
http://mathworld.wolfram.com/Box-and-WhiskerPlot.html 
first one based on the quartiles
